src/templates/kit/screens/TokensScreen.js

```diff
@@ -87,9 +84,37 @@ function Tokens({
     setTokenSymbol(event.target.value.trim().toUpperCase())
   }, [])
 
+  const membersRef = useRef()
+  const [focusLastMemberNext, setFocusLastMemberNext] = useState(false)
+
+  useEffect(() => {
+    if (!focusLastMemberNext || !membersRef.current) {
+      return
+    }
+
+    setFocusLastMemberNext(false)
+
+    // This could be managed in individual MemberField components, but using
+    // the container with a .member class makes it simpler to manage, since we
+    // want to focus in three cases:
+    //   - A new field is being added.
+    //   - A field is being removed.
+    //   - The first field is being emptied.
+    //
+    const elts = membersRef.current.querySelectorAll('.member')
```

Comment:
I remember having a similar problem when working on `AutoComplete`, my first approach used a class to, but I eventually updated to setting a ref array ([here](https://github.com/aragon/aragon-ui/blob/newstyle/src/components/AutoComplete/AutoComplete.js#L31)) and then controlling the index  for AutoComplete's purpose of focusing with keyboard (which in this case would be to focus the input).

Just adding an idea on how to solve this, not really asking for a change.

Reply:
Ah yes, that’s what I was trying to avoid haha. I think it’s simple enough as it is but yes, we might want to handle it by using independent refs in the future.

Reply:
Ok, could we update the class name to something more relevant to the component name? I think it may be safer to do so just in case we follow this approach elsewhere as `.member` is quite generic.

Reply:
`TokensScreen_member`, `Tokens_member` or `Templates_Tokens_member`?